scripts/analyse.sh

```diff
@@ -58,6 +60,47 @@ li_corepart_B15=0.000352885464408
 li_corepart_B95=0.000214330966927
 li_corepart_KUR=0.000334737553995
 
+echo $6 $7
+
+if [ $6 = "ECP"  ]; then
+    corepart=1
+
+elif [ $6 = "ND" ] && [ $7 = "Si"  ]; then
+    corepart=$si_corepart_ND
+
+elif [ $6 = "BN" ] && [ $7 = "Si"  ]; then
+    corepart=$si_corepart_BN
+
+elif [ $6 = "B15" ] && [ $7 = "Si"  ]; then
+    corepart=$si_corepart_B15
+
+elif [ $6 = "B95" ] && [ $7 = "Si"  ]; then
+    corepart=$si_corepart_B95
+
+elif [ $6 = "KUR" ] && [ $7 = "Si"  ]; then
+    corepart=$si_corepart_KUR
+
+elif [ $6 = "ND" ] && [ $7 = "Li"  ]; then
+    corepart=$li_corepart_ND
+
+elif [ $6 = "BN" ] && [ $7 = "Li"  ]; then
+    corepart=$li_corepart_BN
+
+elif [ $6 = "B15" ] && [ $7 = "Li"  ]; then
+    corepart=$li_corepart_B15
+
+elif [ $6 = "B95" ] && [ $7 = "Li"  ]; then
+    corepart=$li_corepart_B95
+
+elif [ $6 = "KUR" ] && [ $7 = "Li"  ]; then
+    corepart=$li_corepart_KUR
+
+else 
+    echo "Core contribution option not found."
+    exit 5
+fi
```

Comment:
It is a good idea to add flexibility to this script, and make the corepart into a cmd-line variable. However, I propose some changes before merging.

The corepart is unique to a given use case, and thus I think we should not include a list to this script. If one wants to add new data to the examples-dir, or use these scripts for some local data, the corepart would need to be set again. If we used your current change, the new corepart-variable would have to be written as a new option to the if-statement every time this script is applied to new data. 

Instead I suggest that we give the corepart as a cmd line option to the analyse.sh-script,  and create a new document to the scripts-folder where we list the coreparts for different materials with different functionals, and document the list appropriately. You can add this list before we merge, or I can do it after the merge. 


```suggestion
corepart=$6
```